### DISCUSSION TITLE
Cannot install on Debian11.5 Gnome

### DISCUSSION THREAD

**Discussion by boris1000** [Schneegans/Fly-Pie#255] [Fly-Pie Q&A] [answered]
Just tried to install on my Debian11.5 - not successful
manual procedure
boris@DELL5070:~$ gnome-extensions install flypie@schneegans.github.com.zip
boris@DELL5070:~$ gnome-extensions enable flypie@schneegans.github.com
Erweiterung »flypie@schneegans.github.com« existiert nicht
boris@DELL5070:~$ 

The first suggestion directly from gnome... when I press install nothing happens
????

**Comment by Schneegans** [comment, marked as answer]
Hi there! Have you restarted you session between installing and enabling the extension? As described in [the guide](https://github.com/Schneegans/Fly-Pie/blob/main/docs/installation.md#b-downloading-a-stable-release)?

**Comment by boris1000** [comment]
Oh don't think so, so when I restart I should be able to enable?
I'll try

Mit freundlichen Grüßen
Boris Stump

Simon Schneegans ***@***.***> schrieb am So., 4. Dez. 2022,
20:10:

**Comment by boris1000** [comment]
Ok, could activate. Hmmm confused - when activated how do I start it,
cannot find it

**Reply by Schneegans** [reply]
Have you read the first-steps guide: https://github.com/Schneegans/Fly-Pie/blob/main/docs/first-steps.md ?
